.github/workflows/benchmarks.yml

```
@@ -4,39 +4,43 @@ on:
   workflow_dispatch:
     inputs:
       runStandalone:
-        description: 'Run the benchmarks against standalone APM Server with Moxy'
+        description: 'Benchmark against standalone APM Server with Moxy'
         required: false
         type: boolean
         default: false
       enableTailSampling:
-        description: 'Enable tail-based sampling on the APM server'
+        description: 'Enable tail-based sampling on APM server'
         required: false
         type: boolean
         default: false
       tailSamplingStorageLimit:
-        description: 'Storage size limit of tail-based sampling on the APM server, defaults to 10GB'
+        description: 'Storage size limit of tail-based sampling on APM server'
         required: false
         type: string
         default: "10GB"
       profile:
-        description: 'The system profile used to run the benchmarks'
+        description: 'System profile for benchmark, e.g. "system-profiles/8GBx1zone.tfvars"'
         required: false
         type: string
       runOnStable:
-        description: 'Run the benchmarks on the latest stable version'
+        description: 'Benchmark on latest stable version'
         required: false
         type: boolean
         default: false
       benchmarkAgents:
-        description: 'Set the number of agents to send data to the APM Server'
+        description: 'Number of benchmark agents sending data to APM Server'
         required: false
         type: string
       benchmarkRun:
-        description: 'Set the expression that matches the benchmark scenarios to run'
+        description: 'Run only benchmark scenarios that match regex, i.e. "go test -run"'
```

```suggestion
        description: 'Run only benchmark scenarios that match regex, e.g. BenchmarkAgentAll
```

sorry I didn't describe it well. It isn't actually `go test -run`, but we mimic `go test -run` in https://github.com/elastic/apm-server/blob/main/systemtest/benchtest/config.go#L48 . But in this case I think it is slightly more helpful to provide an example.